pkg/utils/database/postgres.go

```diff
@@ -168,6 +173,20 @@ func (p Postgres) createDatabase(admin AdminCredentials) error {
 		}
 	}
 
+	if p.DropPublicSchema {
+		if err := p.dropPublicSchema(admin); err != nil {
+			logrus.Errorf("failed dropping the public schema %s", err)
+			return err
+		}
```

Comment:
add here a log if `p.schemas == 0`, that the user self need take care to create schemas.

Reply:
Done